apps/api/src/database/schema/core/defs.ts

```diff
@@ -349,3 +351,47 @@ export const competitionsLeaderboard = pgTable(
     }).onDelete("cascade"),
   ],
 );
+
+/**
+ * Email verification tokens for users and agents
+ */
+export const emailVerificationTokens = pgTable(
+  "email_verification_tokens",
+  {
+    id: uuid().primaryKey().notNull(),
+    userId: uuid("user_id"),
+    agentId: uuid("agent_id"),
+    token: varchar("token", { length: 255 }).notNull().unique(),
```

Comment:
the token looks like this `410cf13e-d40a-4568-929e-58f0f4c4a1f7`, which is length 36. is there any reason why `255` is used here?